assets/tests/components/notificationBellIcon.test.tsx

```diff
@@ -115,4 +138,31 @@ describe("NotificationBellIcon", () => {
       .toJSON()
     expect(tree).toMatchSnapshot()
   })
+
+  test("renders when there are new detour notifications and user is part of DetoursList group", () => {
+    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetoursList])
+
+    const state = { ...initialState, openView: OpenView.None }
```

Comment:
Prefer the state factory if that works!
```suggestion
    const state = stateFactory.build({ openView: OpenView.None })
```

Reply:
[`35dcb01` (#2819)](https://github.com/mbta/skate/pull/2819/commits/35dcb01a1d2f998e202202651a7f46494e5671e0)